ommit bad182d604555235c22fff4200bf419411fa08e4
Author: Marshall Greenblatt <marshall@chromium.org>
Date:   Mon Feb 16 10:56:44 2026 -0800

    Include profile.h unconditionally
    
    Fixes "'Profile' is not defined error" on Windows (around line 141).
    
    Change-Id: I59a5f5697a2aec831a99a71cda53bb256d1f75cd
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/7582413
    Reviewed-by: Eshwar Stalin <estalin@chromium.org>
    Commit-Queue: Marshall Greenblatt <marshall@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1585486}

--- a/chrome/browser/ui/views/frame/system_menu_model_builder.cc
+++ b/chrome/browser/ui/views/frame/system_menu_model_builder.cc
@@ -9,6 +9,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "build/build_config.h"
 #include "chrome/app/chrome_command_ids.h"
+#include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/browser_commands.h"
 #include "chrome/browser/ui/browser_window/public/browser_window_features.h"
 #include "chrome/browser/ui/tabs/features.h"
@@ -31,7 +32,6 @@
 #include "ash/multi_user/multi_user_window_manager.h"
 #include "ash/shell.h"
 #include "chrome/browser/ash/boca/on_task/on_task_locked_controller.h"
-#include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/ash/multi_user/multi_user_util.h"
 #include "chrome/browser/ui/browser_window.h"
 #include "chrome/browser/ui/settings_window_manager_chromeos.h"
